Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -250,7 +250,8 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
     void continueTrackingSelection(PDFDocumentLayout::PageIndex, const WebCore::FloatPoint& pagePoint);
     void stopTrackingSelection();
     void setCurrentSelection(RetainPtr<PDFSelection>&&);
-    void repaintOnSelectionActiveStateChangeIfNeeded(ActiveStateChangeReason);
+    Vector<WebCore::FloatRect> selectionBoundsAcrossDocument(const RetainPtr<PDFSelection>&) const;
```

Comment:
We don't normally pass smart pointers by const reference. Odds are you should be passing a `const PDFSelection*`

Reply:
Changed to `const PDFSelection *` in https://github.com/WebKit/WebKit/commit/9355bda2fca30c7dd4b198ad7569cb367250efcb